applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -180,7 +222,7 @@ int cloud_decode_init(cloud_cmd_cb_t cb);
  * @return 0 if the operation was successful, otherwise a (negative) error code.
  */
 int cloud_encode_digital_twin_data(const struct cloud_channel_data *channel,
-				 struct cloud_msg *output);
+								   struct cloud_msg *output);
```

Comment:
This and below seems off - is it your editor who's messing up? I see some pattern here.

Reply:
fixed.
i'm not sure why these were even modified.  i compare with the clang-formatted output and try to merge only my changes.  probably just user error 😑